src/main/osd/osd_elements.c

```diff
@@ -945,15 +945,27 @@ static void osdBackgroundHorizonSidebars(osdElementParms_t *element)
 static void osdElementLinkQuality(osdElementParms_t *element)
 {
     uint16_t osdLinkQuality = 0;
-    if (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF) { // 0-300
-        osdLinkQuality = rxGetLinkQuality()  / 3.41;
-        tfp_sprintf(element->buff, "%c%3d", SYM_LINK_QUALITY, osdLinkQuality);
-    } else { // 0-9
+    uint8_t osdRFmode = 0;
+    if (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF)
+    { // 0-300
+        osdLinkQuality = rxGetLinkQuality() / 3.41;
+        osdRFmode = rxGetRFmode();
```

Comment:
Change to:
```
const uint8_t osdRfMode = rxGetRfMode();
```
and eliminate the declaration outside the CRSF block.

Reply:
Fixed